dali/test/python/test_operator_nonsilence.py

```diff
@@ -96,7 +94,7 @@ def test_nonsilence_operator():
     window_sizes = [512, 1024]
     reset_intervals = [-1, 2048, 8192]
     references_power = [None, .0003]
-    cutoff_coeffs = [-10, -20, -30]
+    cutoff_coeffs = [-10, -60, -80]
```

Comment:
20 and 30 failed for some samples. 60 is a more typical "silence" definition. Our implementation is different from librosa, so some differences are expected.

Reply:
Do you know why it fails? Difference due to compression?

Reply:
I have no idea. The non-silent region is significantly different to the librosa based one, for those thresholds. That being said, our implementation calculates energy at a much finer granularity (doesn't use windows, I believe)

Reply:
@mzient ?

Reply:
I missed this

> It is because we calculate differently the running average

Yes, we calculate the running average differently. Librosa works with windows, while we have sample granularity, I believe.

Reply:
Understood